src/explore-education-statistics-admin/src/pages/prototypes/components/PrototypeEditableContent.tsx

```diff
@@ -68,43 +73,65 @@ export default class PrototypeEditableContent extends React.Component<
       unsaved: true,
       content: this.temporaryContent,
     });
+
+    const { onContentChange } = this.props;
+    if (onContentChange) {
+      onContentChange(this.temporaryContent);
+    }
   };
 
+  private renderEditableArea(unsaved: boolean, editable?: boolean) {
+    return (
+      // eslint-disable-next-line jsx-a11y/no-static-element-interactions,jsx-a11y/click-events-have-key-events
+      <div
+        className={
+          editable
+            ? `${styles['editable-content']}  ${unsaved ? styles.unsaved : ''}`
+            : ''
+        }
+        onClick={this.setEditing}
+      >
+        <div className={editable ? styles['editable-button'] : ''} />
+        <div ref={this.ref} />
+      </div>
+    );
+  }
+
+  private renderEditor(content: string) {
+    return (
+      <div className={styles['editable-content-editing']}>
```

Comment:
CSS module convention is to use camelCase for class names as this saves on having to do bracket notation

Reply:
done